openshift/template.yaml

```diff
@@ -151,4 +153,9 @@ parameters:
   displayName: Query administration region
   required: false
   name: QUERY_ADMINISTRATION_REGION
-  value: ""
\ No newline at end of file
+  value: ""
+- description: Data replication factor at database level. 
+  displayName: Database Replication Factor
+  required: true
+  name: DB_REPLICATION_FACTOR
```

Comment:
ditto

Reply:
changed it to CASSANDRA_REPLICATION_FACTOR